### PR TITLE
PROTO-1629: sepolia

### PR DESCRIPTION
**Pull request by alecsavvy**
- updates addresses for sepolia network
- updates commands to not reference goerli anymore
- bumps the graph-cli and graph-ts versions as the earlier ones were no longer supported
- bumps the yaml version from 0.0.4 to 0.0.5
- adds many checks that were breaking wasm build on these version updates

functioning graph can be found here: https://thegraph.com/hosted-service/subgraph/audius-infra/audius-network-sepolia
api endpoint: https://api.thegraph.com/subgraphs/name/audius-infra/audius-network-sepolia

### CODE REVIEW

File: src/mappings/governance.ts [hunk 1]
```diff
@@ -92,7 +93,7 @@ export function handleProposalVoteSubmitted(event: ProposalVoteSubmitted): void
   proposalVoteSubmittedEvent.proposal = proposalId
   proposalVoteSubmittedEvent.voter = user.id
   proposalVoteSubmittedEvent.vote = vote.id
-  proposalVoteSubmittedEvent.currentVote = voteType
+  proposalVoteSubmittedEvent.currentVote = voteType === 'Yes' ? voteType : 'No'
```
Comment (raymondjacobson) [inline src/mappings/governance.ts:96]: why was this necessary? type inference?

File: src/mappings/governance.ts [hunk 2]
```diff
@@ -65,6 +65,7 @@ export function handleProposalSubmitted(event: ProposalSubmitted): void {
 export function handleProposalVoteSubmitted(event: ProposalVoteSubmitted): void {
   let proposalId = event.params._proposalId.toString()
   let proposal = Proposal.load(proposalId)
+  if (proposal === null) return
```
Comment (raymondjacobson) [inline src/mappings/governance.ts:68]: if we vote on a proposal that doesn't exist today, does that halt the indexer? i guess it's probably not possible because of the way the smart contract is written, it would prevent that

**Review by raymondjacobson — APPROVED** [review]
wow annoying changes to have to make. nice work!!

File: src/mappings/governance.ts (hunk 1 above)
Comment (alecsavvy) [reply src/mappings/governance.ts:96]: yep!

File: src/mappings/governance.ts (hunk 2 above)
Comment (alecsavvy) [reply src/mappings/governance.ts:68]: I believe returning here would just be a noop on the indexer, if someone proposed a vote on a proposal that doesn't exist then there's nothing to do
